packages/react/kibana_context/render/render_provider.tsx

```diff
@@ -8,33 +8,25 @@
 
 import React, { FC } from 'react';
 
-import { EuiErrorBoundary } from '@elastic/eui';
-import type { I18nStart } from '@kbn/core-i18n-browser';
 import {
-  KibanaThemeProvider,
-  type KibanaThemeProviderProps,
-} from '@kbn/react-kibana-context-theme';
+  KibanaRootContextProvider,
+  type KibanaRootContextProviderProps,
+} from '@kbn/react-kibana-context-root';
 
 /** Props for the KibanaContextProvider */
-export interface KibanaRenderContextProviderProps extends KibanaThemeProviderProps {
-  /** The `I18nStart` API from `CoreStart`. */
-  i18n: I18nStart;
-}
+export type KibanaRenderContextProviderProps = Omit<KibanaRootContextProviderProps, 'globalStyles'>;
 
 /**
  * The `KibanaRenderContextProvider` provides the necessary context for an out-of-current
  * React render, such as using `ReactDOM.render()`.
  */
 export const KibanaRenderContextProvider: FC<KibanaRenderContextProviderProps> = ({
   children,
-  i18n,
   ...props
 }) => {
   return (
-    <i18n.Context>
-      <KibanaThemeProvider {...props}>
-        <EuiErrorBoundary>{children}</EuiErrorBoundary>
-      </KibanaThemeProvider>
-    </i18n.Context>
+    <KibanaRootContextProvider globalStyles={false} {...props}>
```

Comment:
Love this DRYness! I was going to ask you earlier if you thought it would be weird to have the RenderContext reuse the RootContext - glad to know I wasn't totally crazy 😆